### DISCUSSION TITLE
backups, how to save tabs so i can restore them

### DISCUSSION THREAD

**Discussion by gilles007-** [mbnuqw/sidebery#360] [Q&A]
hello, i am interested in automate (each day) a backup of all tabs.
not session restore, i already know how to this
i am interested in backup each tab and it's associated panel
(in the past, i already loose all tabs because i loose windows. now i backup on cloud storage)
snaptshots are located on harddisk ? where ?

**Comment by gilles007-** [comment]
export/import is restoring tabs and panels
even snapshots
but how could i automate this export ?

**Comment by nicklayb** [comment]
I have a similar (the same?) question, I saw a thread where it's said that Panels cannot by synced across devices because of the Firefox Sync's limit but I'm wondering how could I be able to commit them to a repo or something without manual intervention.

What I would want to achieve is:

1. I edit/add/remove a panel from Computer A (I'm having a dev panel for instance that automatically forward language docs there)
2. It's saved to a file in a repo that I can easily commit and push
3. From Computer B; I git pull the repo and my panels gets synced to Computer A's state (I don't care about with tabs themselves, only panel configuration).

I'm trying to get rid of Arc to use only Firefox but the Air traffic control (route url to specific space) is the only thing holding me back, Sidebery does it well with the panels, it's just a matter of synchronizing it across computers now

**Comment by albino1** [comment]
Snapshots are wherever you want to put them, so long as they're in a subfolder of the Firefox download folder.  See Settings -> Snapshots for all of the options.  If you want them to show up in cloud storage then have the folder backup to Dropbox or wherever.  I have it creating 1 snapshot every hour for 48 total Snapshots using the path and format `Sidebery/snapshot-%Y.%M.%D-%h.%m.%s`
